main.yml

```diff
@@ -754,6 +754,10 @@ application-services:
       - id: cos
         title: "Connectors"
         group: overview
+        sub_apps:
```

Comment:
This file is no longer used for navigation and has no effect on it. Can you add an entry to `application-services-navigation.json`?

Reply:
Got it. Actually, this module is conditionally loaded from inside the COS UI.

Reply:
So it does not have an entry in the left nav?

Reply:
yes